common-app-lib/src/cmr/common_app/api_docs.clj

```diff
@@ -151,23 +151,42 @@
   "Defines the footer of the generated documentation page."
   "</div></body></html>")
 
+(defn renderer
```

Comment:
Thanks:
1. I didn't know what else generated docs -- now I do!
1. Turns out that after the refactor, the 3-arity `generate` didn't call to the 4-arity, like it should.

Fixed!